CHANGES.md

```diff
@@ -1,3 +1,12 @@
+# 1.9.0
+
+-   BREAKING: Removed `isRepeatingFormDisabled`. Use the generic `isDisabled`
+    version instead.
```

Comment:
Another breaking thing is that the signature of these isDisabled and so on has changed to take an accessor and not just a field accessor.